### ISSUE TITLE
question re: packaging and releases

### ISSUE THREAD

**Issue by flrgh** [openresty/lua-cjson#96]
We've noticed the [luarocks package of lua-cjson](https://luarocks.org/modules/openresty/lua-cjson) has not been updated in some time:

![image](https://github.com/openresty/lua-cjson/assets/3277009/58762e26-2310-46cf-90c5-4bd79db0c150)

I'm aware the most up-to-date version is bundled with OpenResty itself, but many projects (resty-related and otherwise) take `lua-cjson` as a dependency.

Is it possible to publish an updated package to luarocks? Perhaps this should come with a new tag/release as well, as there has been at least one important bugfix merged since [2.1.0.12](https://github.com/openresty/lua-cjson/releases/tag/2.1.0.12).

**Comment by Tieske** [comment]
Yes, I need this as well, as currently the `lua-resty-ljsonschema` library which depends on `lua-cjson` will install an outdated version on an up-to-date OpenResty system.

**Comment by flrgh** [comment]
@zhuizhuhaomeng sorry to ping you directly, but is there anything that can be done about this?

**Comment by SSE4** [comment]
bump, there is already 2.1.0.13 tag, is it possible to publish new versions to luarocks.org?

**Comment by Tieske** [comment]
Great to see #84 by @hishamhm was merged. So all that's left is a new release and upload to LuaRocks.

Anything we can do to facilitate @zhuizhuhaomeng ?

**Comment by zhuizhuhaomeng** [comment]
@Tieske I didn't use the LuaRocks before?
Would you please prepare the script for LuaRocks?

**Comment by Tieske** [comment]
@zhuizhuhaomeng there is little to it, ensure you have Luarocks installed, and have the Apikey handy, then from the repo do:

		luarocks upload lua-cjson-2.1.0.14-1.rockspec --api-key=<your api-key>

**Comment by Tieske** [comment]
There are also github actions that can automate this process, see https://github.com/lunarmodules/busted/blob/master/.github/workflows/deploy.yml

**Comment by Tieske** [comment]
@zhuizhuhaomeng any update on this? anything you need?

**Comment by zhuizhuhaomeng** [comment]
I don't know how to find the email of the account[ "openresty".](https://luarocks.org/modules/openresty)
Do you know the email of the luarock maintainer?
I would like to contact them to recover the account.

**Comment by Tieske** [comment]
@hishamhm can you provide @zhuizhuhaomeng with the email address of the OpenResty LuaRocks account?

**Comment by Tieske** [comment]
@zhuizhuhaomeng any update on this?
